osx/post_installation.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+echo "Now preparing KA-Lite dependencies.."
+
+SCRIPTPATH=$( cd $(dirname $0) ; pwd -P )
+
+PYRUN_NAME="pyrun-2.7"
+PYRUN_DIR="/Users/Shared/$PYRUN_NAME"
+PYRUN="$PYRUN_DIR/bin/pyrun"
+PYRUN_PIP="$PYRUN_DIR/bin/pip"
+BIN_PATH="$PYRUN_DIR/bin"
+ASSESSMENT_SRC="/Users/Shared/assessment/assessment.zip"
+SHEBANGCHECK_PATH="/Users/Shared/scripts/"
+
+SYMLINK_FILE="/Users/Shared/pyrun-2.7/bin/kalite"
```

Comment:
I suggest you create a variable to hold the `/Users/Shared/` value so we can easily point it somewhere else later.